backend/chalice/api_server/app.py

```diff
@@ -62,7 +64,7 @@ def dispatch(*args, **kwargs):
         with flask_app.test_request_context(
             path=resource_path,
             base_url="https://{}".format(app.current_request.headers["host"]),
-            query_string=app.current_request.query_params,
+            query_string=dict(app.current_request.query_params) if app.current_request.query_params else None,
```

Comment:
Explicitly converting query_param, which is a multidict, into a dict in in the chalice app. Flask had an issues handling the query parameters returned from chalice.  This is an issue that needs further investigations